rails/template.rb

```diff
@@ -22,6 +22,17 @@ def add_template_repository_to_source_path
   end
 end
 
+add_template_repository_to_source_path
+
+# replace sqlite with postgress
+gsub_file 'Gemfile', /gem 'sqlite3'\n/, "gem 'pg'"
+gsub_file 'Gemfile', '# Use sqlite3 as the database for Active Record', ''
+
+# overwrite default sqlite3 database configs with postgresql defaults
+copy_file "config/database.yml", force: true
+gsub_file "config/database.yml", /myapp/, "#{app_name}"
+gsub_file "config/database.yml", /MYAPP/, "#{app_name}.upcase"
```

Comment:
here we replace the myapp with the app_name
even added the special case for the .env constant ;)

Reply:
Ah! Sorry I missed this! You thought about this! Great job! 🥇